src/todolist.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable camelcase */
+export default class TodoList {
+a
+
+constructor() {
+  const savedTasks = JSON.parse(localStorage.getItem('tasks'));
+  if (savedTasks) {
+    this.tasks = savedTasks;
+  } else {
+    this.tasks = [];
+  }
+}
+
+displayTasks() {
+  this.sortAndSave();
+  const check = 'fa-check';
+  const uncheck = 'fa-square-o';
+  // eslint-disable-next-line camelcase
+  const line_through = 'linethrough';
+  const taskSection = document.querySelector('.task-list');
+  taskSection.innerHTML = '';
+  this.tasks.forEach((task) => {
+    const Done = task.completed ? check : uncheck;
+    const Line = task.completed ? line_through : '';
+    const task_temp = `
+        <li class="item d-flex">
+          <div class="task d-flex">
+            <i class="fa ${Done} update-status" job="complete" data="${task.index}"></i>
+            <p class="ptag ${Line}" contenteditable="true"  data="${task.index}">${task.description}</p>
+          </div>
+          <i class="fa fa-trash-o delete-task" job="delete"  data="${task.index}"></i>
+        </li>
+        `;
+    // eslint-disable-next-line camelcase
```

Comment:
- [ ] Please remove these eslint disables and fix any issues that appear. The ESlint warnings usually are a sign of badly organized code so it's best to try to fix these issues.